logger.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+def setup_logger():
+    # Create a logger instance for the app
+    logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)
+
+    # Create a file handler for app.log
+    file_handler = logging.FileHandler('logs/app.log')
```

Comment:
The file name should be an encoding of the information needed to find the respective run. Maybe you will need to return to the logs of a previous run.

Reply:
done